core/src/utils/notifications-service.ts

```diff
@@ -0,0 +1,122 @@
+import Toastify, { Offset } from 'toastify-js'
+
+export enum TypeIcons {
+  'success' = 'check-circle-filled',
+  'error' = 'sparkle-filled',
+  'info' = 'sparkle-filled'
+}
+
+export interface ToastOptions {
+  text?: string | undefined;
```

Comment:
remove undefined and ? from all types here and in the implementation use the type Partial<ToastOptions>